industrial_ci/src/tests/black_check.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+function run_black_check() {
```

Comment:
This needs to be  updated after #558 and to use the new pip function, but I will do so in the next days.

Reply:
That sounds good! Let me know if you want me to fix anything.